Sources/Stripe/Model.swift

```diff
@@ -370,37 +327,22 @@ public struct Plan: Codable, Equatable {
   public var id: Id
   public var interval: Interval
   public var metadata: [String: String]
-  private var name: String? // FIXME: remove
-  private var _nickname: String? // FIXME: remove
```

Comment:
Getting around to cleaning things up now that we've populated the nicknames on the server.